[Teams] add device posture docs (#1844)
cloudflare/cloudflare-docs: products/cloudflare-one/src/content/identity/devices/index.md

---
order: 3
pcx-content-type: how-to
title: Device posture
---

# Device posture attributes

With Teams, you can configure Zero Trust policies that rely on additional signals from endpoint security providers to allow or deny connections to your applications. 

> When device posture checks are configured, users will only be allowed to connect to a protected application when they are using a managed or healthy device, as determined by the endpoint security provider associated to the Zero Trust policy regulating access to the protected application.

Reviewer: ```suggestion
When device posture checks are configured, users can only connect to a protected application if they have a managed or healthy device, as determined by the endpoint security provider associated to the Zero Trust policy regulating access to the protected application.
```
Consider this alternative to simplify the (long) sentence and avoid a few occurrences of passive voice.
